### ISSUE TITLE
Doubling the same type does not trigger MyPy, but should

### ISSUE THREAD

**Issue by socketpair** [python/mypy#13448]
```python
qwe: int | str | int = 45  # MyPy says everything is OK (should not)

int2 = int
asd: int | str | int2  = 45  # MyPy should say OK (and it does)

```

Possibly it's Pylint issue ?

**Comment by AlexWaygood** [comment]
Yeah, this is something for linters to implement, not type checkers. There's nothing _incorrect_ about having duplicate members in a union; it's just bad style.
